### PR TITLE
[IMPROVE] Add modal to close chats when tags/comments are not required

### PR DESCRIPTION
**Pull request by MartinSchoeler**
## Proposed changes (including videos or screenshots)

When neither tags or comments are required to close a livechat, show this modal instead: 
Screen Shot 2021-05-20 at 7 33 19 PM

## Issue(s)

Reiteration of https://github.com/RocketChat/Rocket.Chat/issues/21468

### CODE REVIEW

**Review by rafaelblink — DISMISSED** [review]

File: client/components/Omnichannel/modals/CloseChatModal.js [hunk 1]
```diff
@@ -46,6 +47,19 @@ const CloseChatModal = ({ department = {}, onCancel, onConfirm }) => {
 		);
 	}, [department, tagRequired, t, tags]);
 
+	if (!commentRequired && !tagRequired) {
```
Comment (renatobecker) [inline client/components/Omnichannel/modals/CloseChatModal.js:50]: `tagRequired` is always returning **false**, but I have defined **true** on the department.

**Review by renatobecker — CHANGES_REQUESTED** [review]

**Comment by renatobecker** [comment]
@rafaelblink I was not able to test this PR properly, were you?

File: client/views/room/Header/Omnichannel/QuickActions/QuickActions.tsx [hunk 2]
```diff
@@ -199,6 +199,7 @@ const QuickActions: FC<QuickActionsProps> = ({ room, className }) => {
 	}, [onHoldChat, closeModal, rid, t]);
 
 	const openModal = useMutableCallback((id: string) => {
+		const departmentId = room.departmentId || room.department;
```
Comment (renatobecker) [inline client/views/room/Header/Omnichannel/QuickActions/QuickActions.tsx:202]: `department` field is not a valid field for Omnichannel rooms.

File: definition/IRoom.ts [hunk 3]
```diff
@@ -79,6 +79,7 @@ export interface IOmnichannelRoom extends Omit<IRoom, 'default' | 'featured' | '
 	};
 	onHold?: boolean;
 	departmentId?: string;
+	department?: string;
```
Comment (renatobecker) [inline definition/IRoom.ts:82]: `department` field is not a valid field for Omnichannel rooms.

**Review by renatobecker — CHANGES_REQUESTED** [review]

**Comment by renatobecker** [comment]
The issue related to the required tags feature has not been fixed yet.
The default modal always shows up, even when tags are required.

**Pull request closed without merging**